cmd/yggd/main.go

```diff
@@ -242,13 +252,13 @@ func main() {
 		switch c.String("protocol") {
 		case "mqtt":
 			var err error
-			transporter, err = transport.NewMQTTTransport(ClientID, c.String("server"), tlsConfig, client.DataReceiveHandlerFunc)
+			transporter, err = transport.NewMQTTTransport(ClientID, c.String(cliServer), tlsConfig, client.DataReceiveHandlerFunc)
```

Comment:
If there's a global config variable, the transports can just get the server name from it instead.